### DISCUSSION TITLE
Scheduler not pausing during designated times

### DISCUSSION THREAD

**Discussion by xksteven** [RainbowMiner/RainbowMiner#1682] [Q&A] [answered]
I wanted my miner to only run at night but it seems to be ignoring the schedule. I am running on Ubuntu 20.04. I copied the scheduler below any ideas?

  [
  {
    "Name": "Morning",
    "DayOfWeek": "*",
    "From": "00:00",
    "To": "7:59",
    "PowerPrice": "",
    "Enable": "1",
    "Pause": "0",
    "EnableUpdate": "1",
    "EnableMiningHeatControl": "",
    "MiningHeatControl": "",
    "Algorithm": "",
    "ExcludeAlgorithm": "",
    "CoinSymbol": "",
    "ExcludeCoinSymbol": ""
  },  
  {
    "Name": "Day",
    "DayOfWeek": "*",
    "From": "08:00",
    "To": "19:59",
    "PowerPrice": "",
    "Enable": "0",
    "Pause": "1",
    "EnableUpdate": "1",
    "EnableMiningHeatControl": "",
    "MiningHeatControl": "",
    "Algorithm": "",
    "ExcludeAlgorithm": "",
    "CoinSymbol": "",
    "ExcludeCoinSymbol": ""
  },  
  {
    "Name": "Night",
    "DayOfWeek": "*",
    "From": "20:00",
    "To": "23:59",
    "PowerPrice": "",
    "Enable": "1",
    "Pause": "0",
    "EnableUpdate": "0",
    "EnableMiningHeatControl": "",
    "MiningHeatControl": "",
    "Algorithm": "",
    "ExcludeAlgorithm": "",
    "CoinSymbol": "",
    "ExcludeCoinSymbol": ""
  }
  ]

**Comment by RainbowMiner** [comment, marked as answer]
Two things:
1. input the asterisk into all "DayOfWeek" parameters `"DayOfWeek": "*",`. The "Day" and "Morning" have empty strings, currently
2. enable the "Day" schedule with `"Pause": "1",`
```
{
  "Name": "Day",
  "DayOfWeek": "*",
  "From": "08:00",
  "To": "19:59",
  "PowerPrice": "",
  "Enable": "1",
  "Pause": "1",
  "EnableUpdate": "1",
  "EnableMiningHeatControl": "",
  "MiningHeatControl": "",
  "Algorithm": "",
  "ExcludeAlgorithm": "",
  "CoinSymbol": "",
  "ExcludeCoinSymbol": ""
},
```

Explanation: 
- "Enable" simply tells RainbowMiner to use a specific schedule. It doesn't enable or disable mining.
- "Pause" tells RainbowMiner whether to pause or to mine during the times of an enabled schedule
- "*" in parameter "DayOfWeek" means "on every day of the week"

**Comment by xksteven** [comment]
Thanks!  The "DayOfWeek" fields actually do have the asterisk but markdown interpreted them as to italicize the text.  I tried to format the text as code but then it all appeared on one line and became harder to read.
